src/api/dto/CalloutFormDto.ts

```diff
@@ -72,57 +55,84 @@ abstract class BaseCalloutComponentDto implements BaseCalloutComponentSchema {
   @IsOptional()
   @IsBoolean()
   adminOnly?: boolean;
+
+  // Unused properties
+  [key: string]: unknown;
 }
 
-class ContentCalloutComponentDto extends BaseCalloutComponentDto {
+class CalloutComponentContentDto
+  extends CalloutComponentBaseDto
+  implements CalloutComponentContentSchema
+{
   @Equals(false)
   input!: false;
 
-  @IsIn(["content"])
-  type!: "content";
+  @Equals(CalloutComponentType.CONTENT)
+  type!: CalloutComponentType.CONTENT;
+
+  @IsString()
+  html!: string;
+
+  @IsString()
+  label!: string;
+
+  @IsOptional()
+  @IsObject()
+  validate?: CalloutComponentBaseRules;
+
+  @IsOptional()
+  @IsBoolean()
+  multiple?: boolean;
+
+  @IsOptional()
+  defaultValue?: CalloutResponseAnswer | CalloutResponseAnswer[] | null;
```

Comment:
content components have no inputs and only output html, we still need the properties `validate`, `multiple` and `defaultValue`?

Reply:
I just implemented the types that come from `CalloutComponentBaseSchema`, so I think these probably aren't necesssary but that would need to be updated in beabee-common

Reply:
Than you could use the `CalloutComponentBaseInputSchema` interface for that